screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -59,9 +59,10 @@ const SignUpScreen = ({navigation}) => {
     },
   });
 
```

Comment:
this is an incorrect part of the code move it because if you want to change the formik value you can get the setFieldValues function from formik

setFieldValues("isChecked", the necessary value here)

and delete this const [isChecked, setIsChecked] = useState(false); and useEffect